modules/sxiv/hm.nix

```
@@ -0,0 +1,19 @@
+{ config, lib, ... }:
+
+let
+  colors = config.lib.stylix.colors;
+  fonts = config.stylix.fonts;
+in {
+  options.stylix.targets.sxiv.enable =
+    config.lib.stylix.mkEnableTarget "Sxiv" false;
```

```suggestion
    config.lib.stylix.mkEnableTarget "Sxiv" true;
```

I think it's fine to enable this by default, as we already generate other content in xresources.